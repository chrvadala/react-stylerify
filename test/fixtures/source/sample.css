mystyle{
  color: darkgreen;
  font-size: `return this.number + 'px'`;
  cursor: pointer;

  :hover{
    color: blue;
    text-decoration: underline;
  }
}

anotherstyle {
  font-size: 20px;
}
