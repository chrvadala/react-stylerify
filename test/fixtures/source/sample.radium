mystyle{
  color: darkgreen;
  font-size: `var a = this.number; return a * 2;`;
  cursor: pointer;

  :hover{
    color: blue;
    text-decoration: underline;
  }
}

anotherstyle {
  font-size: 20px;
}
