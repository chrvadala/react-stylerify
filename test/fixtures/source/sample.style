mystyle{
  color: darkgreen;
  font-size: 20px;
  cursor: pointer;

  :hover{
    color: blue;
    text-decoration: underline;
  }
}

anotherstyle {
  font-size: 20px;
}
