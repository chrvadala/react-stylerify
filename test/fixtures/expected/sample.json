{
  "mystyle": {
    "color": "darkgreen",
    "fontSize": "20px",
    "cursor": "pointer",
    ":hover": {
      "color": "blue",
      "textDecoration": "underline"
    }
  },
  "anotherstyle": {
    "fontSize": "20px"
  }
}

